src/core/control/jobs/RenderJob.h

```diff
@@ -37,10 +37,11 @@ class RenderJob: public Job {
     void run() override;
 
 private:
-    /**
-     * Repaint the widget in UI Thread
-     */
-    static void repaintWidget(GtkWidget* widget);
+    void repaintPage();
+
+    void repaintPageArea(double x1, double y1, double x2, double y2);
+
+    void repaintWidgetArea(GtkWidget* widget, int x1, int y1, int x2, int y2);
```

Comment:
Static functions that are only used in the source file should not be declared in the header. See comment above.

```suggestion
```